config/config.go

```diff
@@ -742,6 +742,7 @@ type Route struct {
 	GroupWait      *model.Duration `yaml:"group_wait,omitempty" json:"group_wait,omitempty"`
 	GroupInterval  *model.Duration `yaml:"group_interval,omitempty" json:"group_interval,omitempty"`
 	RepeatInterval *model.Duration `yaml:"repeat_interval,omitempty" json:"repeat_interval,omitempty"`
+	ForwardAlerts  bool            `yaml:"forward_alerts,omitempty" json:"forward_alerts,omitempty"`
```

Comment:
I don't think this should be a stand-alone option - I'd rather have semantics in place using the existing `GroupInterval` and `RepeatInterval`.

Let's finish the discussion on https://github.com/prometheus/alertmanager/issues/2868 before we discuss specifics.